docker-compose.yml

```diff
@@ -22,17 +22,27 @@ services:
     volumes:
       - ./redis:/data
 
-#  es:
-#    restart: always
-#    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.1.3
-#    environment:
-#      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
-#    networks:
-#      - internal_network
-#    healthcheck:
-#      test: ["CMD-SHELL", "curl --silent --fail localhost:9200/_cluster/health || exit 1"]
-#    volumes:
-#      - ./elasticsearch:/usr/share/elasticsearch/data
+#   es:
+#     restart: always
+#     image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.7.1
+#     environment:
+#       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+#       - "cluster.name=es-mastodon"
+#       - "discovery.type=single-node"
```

Comment:
I am upgrading from ``6.8.6`` and this was the only required change for me cause I already set the next line.

Reply:
I've setup the docker-compose.yml per documentation for production.
https://www.elastic.co/guide/en/elasticsearch/reference/6.8/docker.html

Reply:
I mean the ports don't need to be mapped.. I'll remove those